thirdparty/patchmatchnet/models/patchmatch.py

```diff
@@ -140,10 +140,10 @@ def forward(
             height: depth map height,
             width: depth map width,
             depth_sample: sample depth map, in shape of [batch, num_depth, height, width],
-            grid: 2D grid for bilinear gridding, in shape of [batch, len(original_offset)*H, W, 2],
-                    len(original_offset) generally equals to 9, suggesting the 3x3 neighbor grids.
-            depth_min: minimum virtual depth,
-            depth_max: maximum virtual depth,
+            grid: 2D grid for bilinear gridding, in shape of [batch, propagation_neighbors*H, W, 2],

```

Comment:
where is `propagation_neighbors` defined? is this the same as `neighbors` (the input arg)?

maybe prefer:
```python
grid: 2D grid for bilinear gridding, in shape of [batch, neighbors*H, W, 2],
```